source/_integrations/switchbot_cloud.markdown

```
@@ -40,25 +41,139 @@ Please note, device names configured in the SwitchBot app are transferred into H
 
 {% include integrations/config_flow.md %}
 
-## Supported devices
 
```

markdownlint-cli2 (0.17.2)

Multiple consecutive blank lines
Expected: 1; Actual: 2

(MD012, no-multiple-blanks)